index.js

```diff
@@ -28,13 +28,26 @@ function createError (code, message, statusCode = 500, Base = Error) {
     }
 
     this.statusCode = statusCode || undefined
+    this.uriReference = uriReference || undefined
   }
   FastifyError.prototype[Symbol.toStringTag] = 'Error'
 
   FastifyError.prototype.toString = function () {
     return `${this.name} [${this.code}]: ${this.message}`
   }
 
+  FastifyError.prototype.toRFC7807 = function (instance, details) {
+    return {
+      type: this.uriReference || 'about:blank',
+      title: this.name,
+      status: this.statusCode,
+      detail: this.message,
+      instance: instance || '',
```

Comment:
What is this?

Reply:
This would be the place where the error in the fastify route happens. Like you have an ForbiddenError in GET /users/1234 then the instance would be `/users/1234`